Homebrew/Archetypes/Barbarian/Barbarian (Sacred Kin).xml

```diff
@@ -0,0 +1,144 @@
+				<text>The Sacred Kin Spellcasting table shows how many spell slots you have to cast your spells of 1st level and higher. To cast one of these spells, you must expend a slot of the spell's level or higher. You regain all expended spell slots when you finish a long rest.</text>
+				<text>	For example, if you know the 1st-level spell fog cloud and have a 1st-level and a 2nd-level spell slot available, you can cast fog cloud using either slot.</text>
+				<text />
+				<text>Spells Known of 1st Level and Higher</text>
+				<text>You know three 1st-level sorcerer spells of your choice, two of which you must choose from the conjuration and divination spells on the sorcerer spell list.</text>
+				<text>	The Spells Known column of the Sacred Kin Spellcasting table shows when you learn more sorcerer spells of 1st level or higher. Each of these spells must be a conjuration or divination spell of your choice, and must be of a level for which you have spell slots. For instance, when you reach 7th level in this class, you can learn one new spell of 1st or 2nd level.</text>
+				<text>	The spells you learn at 8th, 14th, and 20th level can come from any school of magic on the sorcerer's spell list.</text>
+				<text>	Whenever you gain a level in this class, you can replace one of the sorcerer spells you know with another spell of your choice from the sorcerer spell list. The new spell must be of a level for which you have spell slots, and it must be a conjuration or divination spell, unless you're replacing the spell you gained at 8th, 14th, or 20th level.</text>
+				<text />
+				<text>Spellcasting Ability</text>
+				<text>Constitution is your spellcasting ability for your sorcerer spells, since the power of your magic relies on your ability to harness the magic in your blood. You use your Constitution whenever a spell refers to your spell casting ability. In addition, you use your Constitution modifier when setting the saving throw DC for a spell you cast or when making an attack roll with one.</text>
+				<text>	Spell save DC = 8 + your proficiency bonus + your Constitution modifier</text>
+				<text>	Spell attack modifier = your proficiency bonus + your Constitution modifier</text>
+			</feature>
+			<feature optional="YES">
+				<name>Path of Sacred Kin: Ancestral Origins</name>
+				<text>At 3rd level, when you choose this path, you choose the origin of your ancestor, either celestial, demonic, draconic, or aberrant.</text>
+				<text>Choose one ancestor type from the Ancestral Origins Table. The chosen ancestor associated with your origin is used by features you gain later.</text>
+				<text>You can speak, read, and write the language of your ancestor and can weave their curses and oaths into your battle-cries. Additionally, whenever you make a Charisma check when interacting with a creature of your ancestors' type, your proficiency bonus is doubled if it applies to the check.</text>
+				<text>Ancestor	Damage Type	Language</text>
```

Comment:
THis spacing may not be great, need to check for other items like this

Reply:
looking at druid wild shape, it at least seems consistent, but the rendering is funky, that will also need to be fixed